README.md

```diff
@@ -8,10 +10,8 @@ Follow this guide for help setting it up:
 https://docs.github.com/en/github-ae@latest/get-started/quickstart/set-up-git
 This is the hardest part, and we recommend doing so in club so that we can answer questions while you do this.
 
-After you have set up git, check to see whether you have Python installed on your computer by running in your terminal the following command:
-	> python --version
-If you receive an error, then you need to install python. Do so by running the following command in your terminal:
-	> python
+After you have set up git, check to see whether you have Python installed on your computer by running in your terminal the following command: > python --version
+If you receive an error, then you need to install python. Do so by running the following command in your terminal: > python
```

Comment:
Quotes at the end of the line don't format properly. I understand prettier did this; would it maybe not do it if there were blank lines around the quotes? like this:

```
After you have set up git, do the thing:

	> python --version

If you receive an error, do the thing:

	> python
```